qa/tasks/deepsea.py

```diff
@@ -851,36 +856,64 @@ def _run_orch(self, orch_tuple):
                 cmd_str += ' 2>/dev/null'
         if self.dev_env:
             cmd_str = 'DEV_ENV=true ' + cmd_str
-        try:
-            self.master_remote.run(args=[
-                'sudo', 'bash', '-c', cmd_str,
-                ])
-        except CommandFailedError:
-            self.log.error(anchored(
-                "orchestration {} failed. ".format(orch_spec)
-                + "Here comes journalctl!"
-                ))
-            self.master_remote.run(args=[
-                'sudo',
-                'journalctl',
-                '--all',
-                ])
-            raise
-        self.log.info("orchestration {} completed".format(orch_spec))
+        tries = 0
+        if self.survive_reboots:
```

Comment:
Can't we just use the survive reboots version for every orchestration?
In that case, we could dispense with the `survive_reboots` attribute, the `_detect_reboots` method, and the separate `_exec` method.

Reply:
I wonder if we do this always, then probably it is worth to push it to upstream.

Reply:
> Can't we just use the survive reboots version for every orchestration?
> In that case, we could dispense with the `survive_reboots` attribute, the `_detect_reboots` method, and the separate `_exec` method.

I think we should clearly separate these cases. Imagine a case where machines reboot inadvertently. We would not be able to recognize that properly.

Reply:
OK, I agree, but in that case - in the spirit of "clear separation", I would prefer that `deepsea.orch` take an explicit "allow-reboot: true/false" config parameter, instead of relying on an (undocumented) heuristic that examines "alternative_default".

Reply:
Maybe leave the heuristic, in case someone forgets, but there should be a way to allow reboots without relying on `alternative_default`.

Reply:
Agreed, pushed a version where allow_reboot takes presedence over the heuristic.